WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -161,6 +161,18 @@
       patch :update, id: registration.id, registration: { status: :accepted }
       expect(registration.reload.accepted?).to eq true
     end
+
+    it "can register for their own competition that is not yet visible" do
+      competition.update_column(:showAtAll, false)
+      expect(RegistrationsMailer).to receive(:notify_organizers_of_new_registration).and_call_original
```

Comment:
@jfly in this test there is a post :create to register the organizer for a competition. The registration is created in an instance variable in the create method and passed to notify_organizers_of_new_registration. But how in the test can I access this? @timhabermaas mentioned that we shouldn't use instance variables in these tests moving forward. What's the best way to handle this?